<launch>
  
  <node pkg="ros_pybullet_interface" name="ros_pybullet_interface_node" type="ros_pybullet_interface_node.py">
    <rosparam param="config" file="$(find rpbi_examples)/configs/nsl/config.yaml"/>
  </node>

  <node pkg="rpbi_utils" name="wrench_calibration" type="calibrate_wrench_node.py">
    <remap from="wrench" to="rpbi/kuka_lwr/lwr_arm_7_joint/ft_sensor"/>
    <remap from="wrench/calibrated" to="rpbi/kuka_lwr/lwr_arm_7_joint/ft_sensor/calibrated"/>    
    <param name="max_samples" type="int" value="200"/>
  </node>

  <node pkg="keyboard" name="keyboard" type="keyboard"/>

  <!-- <node pkg="operator_node" name="keyboard_to_joy" type="keyboard_to_joy.py"> -->
  <!--   <rosparam param="config" file="$(find rpbi_examples)/configs/nsl/keyboard_to_joy.yaml"/> -->
  <!-- </node> -->

  <!-- <node pkg="operator_node" name="keyboard_scale" type="isometric_node.py"> -->
  <!--   <rosparam param="config" file="$(find rpbi_examples)/configs/nsl/keyboard_scale.yaml"/> -->
  <!--   <param name="start_on_init" type="bool" value="true"/> -->
  <!--   <remap from="operator_node/signal" to="goal"/> -->
  <!-- </node>   -->

  <node pkg="rpbi_examples" name="nsl" type="nsl.py" output="screen"/>

  <node name="virtuose" pkg="virtuose" type="virtuose_node" output="screen"/>

</launch>
